### PR TITLE
Replace using of namespace cl::sycl:: with sycl::

### PR DESCRIPTION
**Pull request by Alexander-Makaryev**
fixes https://github.com/IntelPython/dpnp/issues/1038

### CODE REVIEW

File: dpnp/backend/kernels/dpnp_krnl_common.cpp [hunk 1]
```diff
@@ -77,18 +77,18 @@ template <typename _KernelNameSpecialization1, typename _KernelNameSpecializatio
 class dpnp_dot_c_kernel;
 
 template <typename _DataType_output, typename _DataType_input1, typename _DataType_input2>
-cl::sycl::event dot(cl::sycl::queue& queue,
+sycl::event dot(sycl::queue& queue,
```
Comment (densmirn) [inline dpnp/backend/kernels/dpnp_krnl_common.cpp:80]: Please align indentations.

File: dpnp/backend/kernels/dpnp_krnl_common.cpp [hunk 2]
```diff
@@ -107,10 +107,10 @@ cl::sycl::event dot(cl::sycl::queue& queue,
 #if LIBSYCL_VERSION_GREATER(5, 3, 0)
         event = queue.submit([&](sycl::handler& cgh) {
             cgh.parallel_for(sycl::range<1>{size},
-                             cl::sycl::reduction(result_out,
+                             sycl::reduction(result_out,
```
Comment (densmirn) [inline dpnp/backend/kernels/dpnp_krnl_common.cpp:110]: the same

File: dpnp/backend/kernels/dpnp_krnl_common.cpp [hunk 3]
```diff
@@ -306,7 +306,7 @@ void dpnp_dot_c(void* result_out,
             // const std::int64_t ldc = result_strides[0] == 1 ? result_strides[1] : result_strides[0];
             const std::int64_t ldc = size_n;
 
-            cl::sycl::event event = mkl_blas_rm::gemm(DPNP_QUEUE,
+            sycl::event event = mkl_blas_rm::gemm(DPNP_QUEUE,
```
Comment (densmirn) [inline dpnp/backend/kernels/dpnp_krnl_common.cpp:309]: the same

File: dpnp/backend/kernels/dpnp_krnl_common.cpp (hunk 1 above)
Comment (Alexander-Makaryev) [reply dpnp/backend/kernels/dpnp_krnl_common.cpp:80]: done

File: dpnp/backend/kernels/dpnp_krnl_common.cpp (hunk 2 above)
Comment (Alexander-Makaryev) [reply dpnp/backend/kernels/dpnp_krnl_common.cpp:110]: done

File: dpnp/backend/kernels/dpnp_krnl_common.cpp (hunk 3 above)
Comment (Alexander-Makaryev) [reply dpnp/backend/kernels/dpnp_krnl_common.cpp:308]: done

**Review by densmirn — APPROVED** [review]

**Pull request merged**